src/metabase/events/schema.clj

```diff
@@ -206,23 +207,34 @@
    [:user-id [:maybe pos-int?]]
    [:model [:or :keyword :string]]])
 
-(mr/def ::data-editing-events
+(mr/def ::nano-id ::common/non-blank-string)
+
+(mr/def ::action-events
   [:map #_{:closed true}
-   (-> [:table_id pos-int?] (with-hydrate :table table-hydrate))
+   [:action :keyword]
+   ;; TODO ... this shouldn't be snake
```

Comment:
I predict that one day there will be a great war within the notifications case about when it's appropriate to use snake keys.